src/container.js

```diff
@@ -63,9 +63,9 @@ function injectProperties(container, fn, instance) {
     for (let property in dependencies) {
       instance[property] = container.get(dependencies[property]);
     }
-  }
-  if (instance.postConstruct !== undefined) {
-    instance.postConstruct();
+    if (instance.postConstruct !== undefined) {
```

Comment:
No problem renaming, but I think this hook is really helpful since it would be the only method the developer can delegate the `Container` to operate on injected properties, since these would not be available in the constructor.

For example, in the following code the hook is helpful otherwise `service` would not be started:
```javascript
class MyClass {

    @inject(Service)
    service;

    constructor() {
       // here this.service is undefined
    }

    afterConstructor() {
        this.service.start();
    }

}
```
The hook will make the developer sure that all instances provided by the `Container` have been correctly initialized with the logic contained in `afterConstructor`; without the hook, `afterConstructor` should be called everywhere `MyClass` is injected.